manifests/fedora-coreos-base.yaml

```
@@ -92,6 +92,8 @@ packages:
   - afterburn
   - afterburn-dracut
   - passwd
+  # Dracut modules
```

Optional. Feel free to resolve/close with no questions asked.

```suggestion
  # Needed by 20live dracut module
```